vendor/engines/split_accounts/app/views/facility_accounts/account_fields/split_accounts/_splits.html.haml

```diff
@@ -0,0 +1,17 @@
+- available_subaccounts = Account.excluding_split_accounts.filter_global_account_types
+= javascript_include_tag "split_account_splits"
+
+%h4 Subaccounts
+
+%div{data: {subaccounts: true}}
+  = f.nested_fields_for :splits, class_name: "SplitAccounts::Split" do |ff|
+    .well
+      = ff.input :subaccount_id, collection: available_subaccounts, wrapper_html: {class: "input-inline"}
+      = ff.input :percent, input_html: {min: 0, max: 100, data: {percent: true}}, wrapper_html: {class: "input-inline"}
+      = ff.input :extra_penny, wrapper_html: {class: "input-inline"}
```

Comment:
The radio wasn't as simple as I had hoped, so I saved it for a future improvement.  I think all of these suggestions are a good idea.  And I was going to circle back to this once I have to reports done.  Does that work for you?